<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateUserInfo</class>
 <widget class="QMainWindow" name="UpdateUserInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="user_groupBox">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>275</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="submit_pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>30</y>
       <width>351</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="username_label">
          <property name="text">
           <string>New UserName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="username_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="password_label">
          <property name="text">
           <string>New password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_lineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="confirm_password_label">
          <property name="text">
           <string>Confirm new password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirm_password_lineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="visibility_label">
          <property name="text">
           <string>Privacy</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="public_radioButton">
            <property name="text">
             <string>public</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="private_radioButton">
            <property name="text">
             <string>private</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
